.browsersync.js

```diff
@@ -18,14 +18,14 @@ module.exports = {
             "port": 8080
         }
     },
-    "files": [ "examples/cdn/*"],
+    "files": [ "examples/bundle/*"],
```

Comment:
Indeed, I was saying it only monitored the examples/cdn folder, that doesn't even exist in the project root (but in website/src isntead). If it's working, then this must be due to what Harry says. Let's remove this file in master?